apps/datafeeder/src/app/presentation/components/upload-data-rules/upload-data-rules.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="text-orange-500">
   <div class="w-20 h-1 bg-orange-500 mb-4"></div>
-  <span class="font-bold">Accepted file formats:</span>
-  <div class="w-64 flex justify-between">
-    <span>- SHP</span>
-    <span>- GeoJSON</span>
+  <span class="font-bold" translate="">{{ 'Accepted file formats' }}:</span>
+  <div
+    *ngFor="let formats of getFormattedFormats()"
+    class="w-64 flex justify-between"
+  >
+    <span>- {{ formats[0] }}</span>
+    <span>- {{ formats[1] }}</span>
   </div>
-  <div class="w-64 flex justify-between">
-    <span>- GeoPackage</span>
-    <span>- Spatialite</span>
-  </div>
-  <span class="font-bold">Maximum size is 30MB</span>
+  <span class="font-bold" translate="">{{
```

Comment:
No it does not work.
Check in `ngx-translate` library how to set up translation with params.